src/OpenTelemetry.SemanticConventions/OpenTelemetry.SemanticConventions.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <Description>OpenTelemetry Semantic Conventions</Description>
+    <PackageTags>$(PackageTags);distributed-tracing</PackageTags>
+    <MinVerTagPrefix>core-</MinVerTagPrefix>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <NoWarn>$(NoWarn),1591,CS0618</NoWarn>
```

Comment:
nit: Not sure if this is just a copy from other projects but I usually suggest adding a comment above explaining what each ignored warning is about and why it's ignored.

Reply:
Yeah it was a copy and paste. I removed it since `1591` is for xml docs on public members, which should be there always. `CS0618` was for Obsolete. This one I don't know if we should keep, since the attributes can be marked obsolete in the spec.. but since here it just defines them and don't reference I'm not sure.